### ISSUE TITLE
Right-click to open gene page doesn't work in Chrome with v3.0.4 beta

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#604]
Right-click to open gene page doesn't work in Chrome version 66.0.3359.170 with v3.0.4 beta.  I suspect this may have something to do with pop-up blockers.  When I tested in Firefox, I got a banner asking me whether to allow the site to open a pop-up window.  Maybe my settings on Chrome are just blocking them.

**Comment by kdahlquist** [comment]
At the meeting today, I'll bring my laptop and check my settings against one of yours to see which setting might be interfering.

**Comment by johnllopez616** [comment]
Will examine on Safari when @mihirsamdarshi arrives today.

**Comment by johnllopez616** [comment]
The issue seemingly resolved itself on Safari somehow, and I believe the issues on Chrome and Firefox relate directly to settings. When you get back from class I would like to compare the settings on our computers to ensure this.

**Comment by kdahlquist** [comment]
@kdahlquist confirmed that this is no longer an issue.  The message about allowing pop-ups on Chrome was so small I didn't see it at first.  Closing this one.  @mihirsamdarshi has requested to open another issue to bypass the pop-up blockers entirely.

**Comment by dondi** [comment]
Yes @kdahlquist I was actually meaning to bring this up at the last meeting—the practice of opening a new tab or window in a web app is beginning to yield (for security reasons) to having in-window modals or popovers. Unfortunately, doing those in a practical manner does require moving to the newer tech stack that we did talk about last Monday.

New tabs and windows are not blocked by the browser if they were already pre-existing in the page. However, by nature, our genes cannot be that way. So we may need to tolerate popup blockers through version 3.x, and target not needing to deal with them come version 4.x.

**Comment by johnllopez616** [comment]
Would using the `window.open` be a fair compromise to this issue? I have modified the left-clicking functionality to do this instead of creating a temporary link as before.

You may see the changes made in this Codepen:
https://codepen.io/anon/pen/jxQZxp?editors=1010
